.travis.yml

```diff
@@ -23,8 +23,10 @@ before_install:
   - pip install -U pip
   - pip install codecov
 install:
+  - pip install -r requirements-dev.txt
   - pip install  ".[test]" .
```

Comment:
Can be removed, as we're using the `requirements-dev.txt`

Reply:
removed with 9209785b76efe21c56eb3e7df135d09c0028e3ad